tests/pretty_format_json_test.py

```diff
@@ -48,6 +51,17 @@ def test_tab_pretty_format_json(filename, expected_retval):
     assert ret == expected_retval
 
 
+@pytest.mark.parametrize(('filename', 'expected_retval'), (
+    ('not_pretty_formatted_json.json', 1),
+    ('unsorted_pretty_formatted_json.json', 1),
+    ('non_ascii_pretty_formatted_json.json', 0),
+    ('pretty_formatted_json.json', 0),
+))
+def test_non_ascii_pretty_format_json(filename, expected_retval):
```

Comment:
I wouldn't bother to parametrize this, you're really only trying to demonstrate the one file :shrugs: